docs/userGuide/plugins/googleAnalytics.mbdf

```diff
@@ -0,0 +1,20 @@
+#### `Google Analytics`: Enhancing site with Google Analytics
+
+This plugin allows your web pages to be enhanced with [google analytics](https://analytics.google.com/analytics/web/#/) to track, analyse and improve your content.
+
+To add Google Analytics plugin, simply add the following to `site.json`:
```

Comment:
Should there be a basic description on how to get the tracking ID? They can be told to visit this link: https://support.google.com/analytics/answer/1008080.

Reply:
In fact, let's follow the format of Algolia, with a table for the `trackingID` parameter.